App/VSTS/CopyButton.tsx

```diff
@@ -0,0 +1,35 @@
+/// <reference path="../../office.d.ts" />
+import * as React from 'react';
+
+
+interface ICopyButtonProps {
+  workItemHyperlink: string;
+}
+
+export class CopyButton extends React.Component<ICopyButtonProps, {}> {
+
+  public render(): React.ReactElement<{}> {
+    console.log('got to copybutton');
+    document.addEventListener('copy', this.setClipboardHandler);
```

Comment:
Do we need to do the add event handler on render or can we move it to one of the standard methods like component mount or update?